Source/Initialization/WarpXInitData.cpp

```diff
@@ -1228,3 +1282,152 @@ void WarpX::CheckKnownIssues()
                 ablastr::warn_manager::WarnPriority::low);
         }
 }
+
+#if defined(WARPX_USE_OPENPMD) && !defined(WARPX_DIM_1D_Z)
+void
+WarpX::ReadExternalFieldsFromFile (std::string read_fields_from_path, MultiFab* mf,
```

Comment:
This should really be `ReadExternalField[FromFile]` or `ReadExternalFieldComponent[FromFile]`, since it only operates on one field and component at a time.